docs/articles/vs-test-adapter/Adapter-License.md

```diff
@@ -1,22 +1,23 @@
 # License - NUnit and NUnit3 Test Adapters for Visual Studio
 
-## Copyright © 2015-2016 Charlie Poole
+## MIT License
```

Comment:
Can you please link to a discussion or agreement somewhere to change the licensing and add you to the copyright? Just want to make sure we're doing this by the book.

Reply:
I just noticed this recently, and it is probably an oversight from years back.  @CharliePoole can confirm.  The copyright for the adapter is by Charlie Poole 2011 and me 2014, and the license is MIT.  This can be seen in the NUnit3Adapter repo.

Reply:
@SeanKilleen The documentation copy of the license is secondary. If it's provided, it must always be a copy of the LICENSE file provided with the software, which is where the history lives. My guess is that some old file had not been updated at the point where you converted the docs to docfx.

I modified the license to include Terje as a co-owner of the software in 2017. See https://github.com/nunit/nunit3-vs-adapter/commit/21624d1348d2274e094914be5e98ef312cd3523b

@OsirisTerje You should not be updating my dates in the header after I stopped contributing. You could also reasonably decide that 4.0 is now a new work of your creation and remove my name. That's what I did on the release of NUnit 3 (framework, console and GUI) after verifying that virtually no remaining code had been contributed by others. I also added a notice file that mentioned the original creators - not required by MIT but it seemed only polite. It's a question of your judgement in my opinion.

In case anyone should wonder, Terje and I are the sole copyright holders for the adapters. The Core Team has no involvement in the decision, although an individual member could challenge the copyright if they felt it was wrong.